src/Admin/Jobs/JobsHostedService.cs

```diff
@@ -55,13 +55,19 @@ public override async Task StartAsync(CancellationToken cancellationToken)
                 .StartNow()
                 .WithCronSchedule("0 0 0 ? * SUN", x => x.InTimeZone(timeZone))
                 .Build();
+            var everyDayAtMidnightUtc = TriggerBuilder.Create()
+                .WithIdentity("EveryDayAtMidnightUtc")
+                .StartNow()
+                .WithCronSchedule("0 0 0 * * ?")
+                .Build();
 
             var jobs = new List<Tuple<Type, ITrigger>>
             {
                 new Tuple<Type, ITrigger>(typeof(DeleteSendsJob), everyFiveMinutesTrigger),
                 new Tuple<Type, ITrigger>(typeof(DatabaseExpiredGrantsJob), everyFridayAt10pmTrigger),
                 new Tuple<Type, ITrigger>(typeof(DatabaseUpdateStatisticsJob), everySaturdayAtMidnightTrigger),
-                new Tuple<Type, ITrigger>(typeof(DatabaseRebuildlIndexesJob), everySundayAtMidnightTrigger)
+                new Tuple<Type, ITrigger>(typeof(DatabaseRebuildlIndexesJob), everySundayAtMidnightTrigger),
+                new Tuple<Type, ITrigger>(typeof(DeleteCiphersJob), everyDayAtMidnightUtc)
```

Comment:
Are the other jobs not based on UTC time? This is the only one specifying such in the trigger name.

Reply:
The others are in Eastern time zone. This is the only UTC one, which is why I labeled it this way